cmd/mtaBuild.go

```diff
@@ -314,6 +315,21 @@ func handleDefaultNpmRegistry(config mtaBuildOptions, e execRunner) error {
 	return nil
 }
 
+func handleSAPNpmRegistry(config mtaBuildOptions, e execRunner) error {
+
+	if len(config.SapNpmRegistry) > 0 {
+
+		log.Entry().Debugf("Setting SAP npm registry to \"%s\"", config.SapNpmRegistry)
+		if err := e.RunExecutable("npm", "config", "set", "@sap:registry", config.SapNpmRegistry); err != nil {
```

Comment:
How do we ensure that this is applicable for all modules in the mta? Set this with `--global`?

Reply:
good catch. It is not nice to alter the user-owned config, but I'm not aware of a better option. Also I think we need to do this for the "default" registry..

Reply:
no, actually I think this is not true. `npm config set` changes it in my home dir, `--global` would set it in `$PREFIX/etc/npmrc`.